openshift-hack/images/os/Dockerfile

```diff
@@ -21,3 +21,5 @@ RUN set -x && yum install -y ostree rpm-ostree yum-utils selinux-policy-targeted
 
 FROM scratch
 COPY --from=build /srv/ /srv/
+LABEL io.openshift.build.version-display-names="machine-os=Red Hat Enterprise Linux CoreOS (testing)" \
+      io.openshift.build.versions="machine-os=1.2.3-testing-machine-os"
```

Comment:
support for SemVer values [here][1], so my string being SemVer means shouldn't break any in-cluster assumptions. 
 And the fact that it's clearly a dummy/test value will hopefully keep human readers from assuming that the resulting build-for-CI images are official RHCOS releases.

[1]: https://github.com/openshift/oc/pull/209#issuecomment-564876535